app/src/main/java/com/x8bit/bitwarden/data/vault/manager/VaultLockManagerImpl.kt

```diff
@@ -70,12 +69,12 @@ class VaultLockManagerImpl(
     private val userLogoutManager: UserLogoutManager,
     private val trustedDeviceManager: TrustedDeviceManager,
     dispatcherManager: DispatcherManager,
-    private val elapsedRealtimeMillisProvider: () -> Long = { SystemClock.elapsedRealtime() },
 ) : VaultLockManager {
     private val unconfinedScope = CoroutineScope(dispatcherManager.unconfined)
 
+    private val userIdTimerJobMap = mutableMapOf<String, Job>()
```

Comment:
Should we have documentation for what this does?

Reply:
Sure